docusaurus/docs-internals/brizy-editor/introduction.md

```diff
@@ -3,3 +3,35 @@ sidebar_position: 1
 ---
 
 # Introduction
+
+The environment where users design pages and oversee site settings is referred to as the `editor`. This environment comprises two key segments: the `preview` and the `editor`. Within each segment, there are additional areas that third-party developers can expand.
+
+---
+
+### Using Controls 
+
+Brizy controls are UI components designed to help build toolbars and gather user data. They come in various types to accommodate different needs:
+
+- [Usage](/docs-internals/using-controls/usage/)
+- [Regular](/docs-internals/using-controls/regular-control/)
+- [Responsive](/docs-internals/using-controls/responsive-control/)
+- [Group](/docs-internals/using-controls/control-group/)
+- [Popover](/docs-internals/using-controls/control-popover/)
+- [Tabs](/docs-internals/using-controls/control-tabs/)
+
+### Editor Controls 
+
+The Brizy Editor control allows users to edit content directly within the page builder interface, providing a seamless and intuitive experience for content customization. This control is a key component in Brizy, enabling users to make real-time adjustments to text, images, and other elements without needing to delve into the code.
+- [Editor](/docs-internals/editor-controls/introduction/)
+
+### CSS Controls
+
+Brizy's CSS mechanism allows users to dynamically customize the appearance of website elements without coding. Users can adjust CSS properties such as colors, fonts, margins, and more through an intuitive interface.
```

Comment:
this text is copied

there is no need to copy any text, just write a short description what each module contains